doc/reflection-milestone2.md

```diff
@@ -1,3 +1,29 @@
-# author: Jennifer
+# Reflection - Milestone 2
+Authors: Christopher Alexander, Jennifer Hoang, Wenxin Xiang, and Michelle Wang
 
-Content of reflection for milestone2.
+### What has been implemented?
+Our group has added 4 interactive plots to our dashboard that describe popular artists and songs on Spotify.
+The 3 plots in the "Artists/Genre" tab allow users to explore top artists, artist popularity trends over various album releases,
+and the overall popularity distribution of their songs. We have implemented two widgets on this page, 
+which allow users to select the genre for the top artists plot from a dropdown menu.
+Users can also select specific artists of interest from a dropdown menu to explore their trend and distribution plots in detail.
+
+In the second "Song Characteristics" tab, users can compare musical features between popular and non-popular songs.
+The musical feature displayed on the X-axis, as well as the genre of music, can be selected from the dropdown menu.
+
+Overall, our dashboard is able to provide a quick glimpse at the music trends across artists and genres on Spotify in 2020.
+
```

Comment:
This suffices the need of our target users, content managers at Spotify, to get music tracks insights visually and quickly to support their data driven decision making process.